packages/patternfly-4/content/design-guidelines/usage-and-behavior/modal/design.md

```diff
@@ -86,10 +86,10 @@ See our [content guidelines](/design-guidelines/content/) for additional guidanc
 
 | Icon  | Use Case(s) | Usage |
 | ------------- | ------------- | ------------- |
-| ![exclamation-triangle](img/exclamation-triangle.svg) exclamation-triangle  | **Warning:** Caution/ Warning | Use on confirmation dialogs or passive dialogs to indicate a higher level of urgency and importance. |
-| ![exclamation-circle](img/exclamation-circle.svg) exclamation-circle  | **Critical Warning:** Information will be deleted/permanent action  | Use on confirmation dialogs or passive dialogs to indicate the highest level of urgency and importance. |
-| ![times-circle](img/times-circle.svg) times-circle  | **Error:** Alert the user that there has been a critical failure/error  | Use on error dialogs to indicate a problem. |
-| ![info-circle](img/info-circle.svg) info-circle  | **Acknowledgement:** Informs the user of an action or result  | Use on confirmation or passive dialogs to indicate a lower level of urgency. |
+| <i class="fas fa-exclamation-triangle"></i> | **Warning:** Caution/ Warning | Use on confirmation dialogs or passive dialogs to indicate a higher level of urgency and importance. |
```

Comment:
These appear as intended on the site, but are empty when viewing as pure markdown. I'm not sure if that matters or not?

Reply:
I guess that makes sense that it only renders for the website, i think its fine